### ISSUE TITLE
Confusion on Validation Dataset

### ISSUE THREAD

**Issue by tengwang0318** [ilnehc96/HKU-DASC7606-A1#11]
I am confused that the validation loss hovers at some level(0.6-0.75), but the mAP is growing up constantly(from 0.3 to 0.48). I am not good at CV and don't change the model structure. I may think the process is already overfitting based on validation loss if I am doing some NLP, RL tasks. But after checking the result of mAP, feel quite surpised. 

Is that because the model don't tackle with difficult images and only deal with exactly more easy images?
